test/util/conditional.spec.js

```diff
@@ -0,0 +1,157 @@
+const expect = require('chai').expect;
```

Comment:
The formatting in all of these tests is wonky, come talk to me when you get a minute @krackerman